info.md

```diff
@@ -1,9 +1,17 @@
-<img src="https://github.com/jcwillox/button-entity-row/blob/master/examples/example-gif.gif?raw=true" width="400px">
-<img src="https://github.com/jcwillox/button-entity-row/blob/master/examples/example-1.png?raw=true" width="400px">
-<img src="https://github.com/jcwillox/button-entity-row/blob/master/examples/example-3.png?raw=true" width="400px">
+{% if installed and version_installed.replace("v", "").replace(".","") | int <= 20  %}
```

Comment:
How does that works? I've never seen this before in a markdown file

Reply:
It's not exactly a markdown file, its HACS version of markdown, which supports Jinja templating, it's pretty cool though [hacs docs](https://hacs.netlify.com/developer/general/#templates).